### PR TITLE
Update dependencies

### LINKED ISSUE #3: Hazelcast exporter creates tons of connections to hazelcast nodes until limit is reached
`hazelcast_exporter` opens new connection every few seconds to each Hazelcast node in cluster. All of them are marked as established. As a result, after few minutes, `nofile` limit is reached and service stops to work and respond.

We have `nofile` limit set to `1024`. `2607` is PID of hazelcast_exporter:
```
> lsof -p 2607 | wc -l
1028
```

Few of log lines:
```
Jan 11 08:32:12 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:12 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 40ms
Jan 11 08:32:12 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:12 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 80ms
Jan 11 08:32:12 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:12 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 160ms
Jan 11 08:32:12 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:12 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 320ms
Jan 11 08:32:12 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:12 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 640ms
Jan 11 08:32:13 hazelcast-1 hazelcast_exporter[14587]: could not connect to any addresses
Jan 11 08:32:13 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:13 New State :  SHUTTING_DOWN
Jan 11 08:32:13 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:13 New State :  SHUTDOWN
Jan 11 08:32:13 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:13 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 1s
Jan 11 08:32:14 hazelcast-1 hazelcast_exporter[14587]: 2019/01/11 08:32:14 http: Accept error: accept tcp [::]:8080: accept4: too many open files; retrying in 1s
```

Setting higher limit is only an workaround I think.

### PR DESCRIPTION
**Pull request by roidelapluie**
Fix #3

### CODE REVIEW

**Comment by roidelapluie** [comment]
@zbikmarc

**Comment by zbikmarc** [comment]
I was also working on it xD

**Comment by roidelapluie** [comment]
I am curious if you see https://github.com/hazelcast/hazelcast-go-client/issues/412 too

File: Gopkg.lock [hunk 1]
```diff
@@ -3,108 +3,151 @@
...
+    "internal/util/timeutil",
+    "internal/util/versionutil",
+    "security",
     "serialization",
-    "serialization/classdef"
+    "serialization/classdef",
   ]
-  revision = "57b5e47611af979ddc50b1f628bb38c7d44db0d4"
+  pruneopts = "UT"
+  revision = "3560be65ca6037e3adf8cf5cd222dabf0f980530"
```
Comment (zbikmarc) [inline Gopkg.lock]: Set revision to `4e0c3ccf8841745e674e2419f3dcb20de0a60098` please. This contains latest fixes for connections not being closed.

**Comment by roidelapluie** [comment]
better?

**Comment by zbikmarc** [comment]
Awesome!

Regarding https://github.com/hazelcast/hazelcast-go-client/issues/412 - I did not see this yet.

**Pull request merged**